.github/workflows/test-integrations.yml

```diff
@@ -13,8 +13,13 @@ jobs:
       SECURE_INTEGRATION: false
     strategy:
       matrix:
-        cluster-type: ['opendistro', 'opensearch']
-        secured: ["true", "false"]
+        entry:
+          - { cluster: 'opendistro', version: '1.13.3', secured: 'true' }
+          - { cluster: 'opendistro', version: '1.13.3', secured: 'false' }
+          - { cluster: 'opensearch', version: '1.2.4', secured: 'true' }
```

Comment:
Do we want to add all other 1.x versions from 1.0.0 to 1.2.3?

Reply:
https://github.com/opensearch-project/opensearch-rs/pull/44/commits/277f2b6d4a7be5302adf52120a711121ebb2e09f